recipes/simde/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "simde" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simd-everywhere/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 366d5e9a342c30f1e40d1234656fb49af5ee35590aaf53b3c79b2afb906ed4c8
+
+build:
```

Comment:
If I understand correctly, this is a header-only library? Is there any dependency on the compiler that was used to build this package? If not, you could add something like the following

```
  ignore_run_exports_from:
    # Since this package does not ship any compiled code, we do not need any
    # compatibility with the compiler used at runtime.
    - {{ compiler('cxx') }}
```

Reply:
Done.